index.js

```diff
@@ -461,6 +461,44 @@ class BitbucketScm extends Scm {
         });
     }
 
+   /**
+    * Checkout the source code from a repository; resolves as an object with checkout commands
+    * @method getCheckoutCommand
+    * @param  {Object}    config
+    * @param  {String}    config.branch        Pipeline branch
+    * @param  {String}    config.host          Scm host to checkout source code from
+    * @param  {String}    config.org           Scm org name
+    * @param  {String}    config.repo          Scm repo name
+    * @param  {String}    config.sha           Commit sha
+    * @param  {String}    [config.prRef]       PR reference (can be a PR branch or reference)
+    * @return {Promise}
+    */
+    _getCheckoutCommand(config) {
+        const checkoutUrl = `https://${config.host}/${config.org}/${config.repo}`;
+        const checkoutRef = config.prRef ? config.branch : config.sha;
+        let command;
+
+        // Git clone
+        command = `echo Cloning ${checkoutUrl}, on branch ${config.branch}`;
+        command += ` && git clone --quiet --progress --branch ${config.branch} ${checkoutUrl}`;
```

Comment:
See my comment in scm-github: https://github.com/screwdriver-cd/scm-github/pull/39#discussion_r87520184